docker-compose-proxy.yml

```diff
@@ -0,0 +1,26 @@
+version: '3.3'
```

Comment:
I really don't like the idea of multiple Docker Compose files, it makes the original intent of just using `docker-compose up` a bit more difficult when differentiating between environments.

Although I heard from a previous co-worker that at your talk, you mentioned Docker Compose eventually going away and/or only used as a development tool?